pset-iterators.js

```diff
@@ -183,7 +210,13 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 7:')
 
 // Add your code below this line
-
+const shortTermPresidents = presidents.filter((allTerm) => {
```

Comment:
@cbrooks2019 nice work, code was well-structured and easy to follow

By using `.parseInt()` on the entire date strings (i.e. `"1921-03-04"`), you were able to extract out the year (1921) as an integer to use for your comparison. 

.